Adyen/Model/StorePaymentMethodAware.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+@_spi(AdyenInternal)
+public protocol StorePaymentMethodFieldAware {
```

Comment:
Do we need to add /// comments to _spi classes / properties ?

Reply:
Not necessarily but good catch, I'll add them.